STM32/DC/Core/Src/DCBoard.c

```diff
@@ -213,33 +223,36 @@ static void actuatePins(ActuationInfo actuationInfo)
 {
     if (!actuationInfo.isInputValid)
     {
+        HALundefined("Invalid input");
+        return;
+    }
+    else if (actuationInfo.pin < 0 || actuationInfo.pin >= ACTUATIONPORTS) {
```

Comment:
It seems odd to me that this isn't being captured in the isInputValid check above.

Reply:
There is no check. I'll remove that variable.